### ISSUE TITLE
Can't resize while uploading image

### ISSUE THREAD

**Issue by ronggur** [whitecube/nova-flexible-content#63]
Hi All, Thanks for this amazing package.

I have a question, I tried using ->store method for Image field to resize, but nothing happened. FYI: I able to use this method in default Image nova field.

Am I missing something here?

Thanks

```
->store(function (Request $request) {
    $image = doin_resize_function();

    return [
        'image' => $image
    ];
})
```

**Comment by Nyratas** [comment]
Hi @ronggur,

I'll have to give this a closer look, but I think it could automatically get resolved with the next release thanks to 26ab122b4622209b43a8c8004de09e0cc4615b83. Not 100% sure, we'll have to test this use case soon.

**Comment by ronggur** [comment]
Awesome, thanks @Nyratas

**Comment by Nyratas** [comment]
Hi @ronggur,

We just released v0.1.10, could you update and tell me if it solved this issue? Thanks!

**Comment by ronggur** [comment]
Hey @Nyratas it's working flawlessly, Thank you!
